app/styles/variables.less

```diff
@@ -1,4 +1,4 @@
 // copied from origin-web-console
 @panel-shaded: #f2f2f2;
 
-@navbar-height: 69px;
+@navbar-height: 72px;
```

Comment:
Can we punt on this?  Given this only affects the demo app, I'd rather spend cycles getting unstyled parts styled.